src/main/java/org/broadinstitute/hellbender/tools/spark/sv/sga/ExternalCommandlineProgramModule.java

```diff
@@ -1,160 +0,0 @@
-package org.broadinstitute.hellbender.tools.spark.sv.sga;
-
-import com.google.common.base.Throwables;
-import org.apache.commons.io.FileUtils;
-
-import java.io.File;
-import java.io.IOException;
-import java.io.Serializable;
-import java.nio.charset.Charset;
-import java.nio.file.Files;
-import java.nio.file.Path;
-import java.util.List;
-
-/**
- * A class for experimenting with cmd line programs (such as bwa and sga) that's not part of GATK (or has Java bindings yet).
- */
-abstract class ExternalCommandlineProgramModule {
```

Comment:
Do you want to keep this one around? It's not SGA specific and seems like it could be useful sometime in the future.

Reply:
Do you think it would be reasonable, to keep around the `SGAModule` as well, for illustration purpose?

Reply:
I'll mark it `@Deprecated` as is already the case.